rllib/utils/exploration/epsilon_greedy.py

```diff
@@ -94,7 +94,7 @@ def _get_tf_exploration_action_op(self, q_values, explore, timestep):
 
         chose_random = tf.random_uniform(
             tf.stack([batch_size]),
-            minval=0, maxval=1, dtype=epsilon.dtype) \
+            minval=0, maxval=1, dtype=tf.float32) \
```

Comment:
IIRC this was crashing, though I forget how to reproduce it exactly.

Reply:
Hmm, ok.